docs/Helm.md

```diff
@@ -0,0 +1,106 @@
+# Splunk Operator Helm Installation
+
+## Splunk Operator Helm chart Repository
+
+Add the Splunk Operator and Enterprise charts to your Helm repository.
+
+```
+helm repo add splunk https://splunk.github.io/splunk-operator/
+helm repo update
+```
+
+The ```splunk``` chart repository contains the ```splunk/splunk-operator``` chart to deploy the Operator and the ```splunk/splunk-enterprise``` chart to deploy Splunk Custom Resources.
+
+Helm provides a long list of commands to manage your deployment, we'll be going over a few useful ones in the sections to come. You can learn more about supported commands [here](https://helm.sh/docs/helm/helm/).
+
+## Splunk Operator deployments
+
+To install the Splunk Operator:
+
+```
+helm install <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+To upgrade the operator, specify override configurations through the CLI or a new file:
+
+```
+helm upgrade --set splunkOperator.clusterWideAccess=false <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+Here we've upgraded the operator by revoking cluster-wide access.
+
+To rollback the operator to a previous release:
+
+```
+helm rollback <RELEASE_NAME> <REVISION_NUMBER> -n <RELEASE_NAMESPACE>
+```
+
+To clean-up resources and uninstall the operator:
+
+```
+helm uninstall <RELEASE_NAME> -n <RELEASE_NAMESPACE>
+```
+
+### Configuring Splunk Operator deployments
+
+There are a couple ways you can configure your operator deployment
+
+1. Using a ```new_values.yaml``` file to override default values (Recommended)
```

Comment:
Also, I think, it will be nice to have, an example deployment with the helm commands with some non-defaults, which will be very good reference for someone new to the helm.